cli/azd/cmd/templates.go

```diff
@@ -81,10 +83,10 @@ func (tl *templatesListAction) Run(ctx context.Context) error {
 		return err
 	}
 
-	templateList := make([]templates.Template, 0, len(templateSet))
-	for _, template := range templateSet {
-		templateList = append(templateList, template)
-	}
+	templateList := maps.Values(templateSet)
+	slices.SortFunc(templateList, func(a, b templates.Template) bool {
+		return a.Name < b.Name
+	})
```

Comment:
Any specific reason to use `slices.SortFunc(...)` over `sort.Slice(...)` ?

Reply:
`sort.Slice` looks like the old-non-generic implementation. `slices.SortFunc` is the newer generic-based implementation. They both work the same for our use case.